apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1118 @@
+                  onChange={(e) => setNewProperty(prev => ({ ...prev, rules: e.target.value }))}
+                  className="w-full px-3 py-2 border dark:text-white border-gray-300 rounded-lg focus:ring-0 bg-transparent focus:ring-blue-500 focus:border-transparent"
+                  placeholder="No smoking, No pets, Quiet hours after 10 PM, etc."
+                />
+              </div>
+
+              {/* Photos */}
+              <div className="pb-6">
+                <h3 className="text-lg font-semibold dark:text-white text-gray-900 mb-4">Photos</h3>
+                <div className="border-2 border-dashed border-gray-300 rounded-lg p-8 text-center">
+                  <div className="text-gray-500">
+                    <svg className="mx-auto h-12 w-12 text-gray-400" stroke="currentColor" fill="none" viewBox="0 0 48 48">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)